public/scripts/macros.js

```diff
@@ -464,6 +464,7 @@ export function evaluateMacros(content, env) {
     content = content.replace(/{{firstIncludedMessageId}}/gi, () => String(getFirstIncludedMessageId() ?? ''));
     content = content.replace(/{{lastSwipeId}}/gi, () => String(getLastSwipeId() ?? ''));
     content = content.replace(/{{currentSwipeId}}/gi, () => String(getCurrentSwipeId() ?? ''));
+    content = content.replace(/{{reverse\:(.+?)}}/gi, (_, str) => str.split('').reverse().join(''))
```

Comment:
This is not a proper way to reverse a string as this will split runes with multiple code points, e.g.

`'Emoji😀' => '\uDE00\uD83DijomE'`

A proper way to do it would be

```js
Array.from('Emoji😀').reverse().join('')
'😀ijomE'
```